web/controllers/certificateController/index.ts

```diff
@@ -494,18 +492,17 @@ function createPDFBinary(certificate: ParticipationCertificate, link: string, la
         ONGOING: certificate.ongoingLessons,
         SIGNATURE_PARENT: certificate.signatureParent?.toString("utf-8"),
         SIGNATURE_PUPIL: certificate.signaturePupil?.toString("utf-8"),
-        SINGATURE_LOCATION: certificate.signatureLocation,
-        SIGNATURE_DATE: certificate.signatureDate
+        SIGNATURE_LOCATION: certificate.signatureLocation,
+        SIGNATURE_DATE: certificate.signatureDate && moment(certificate.signatureDate).format("D.M.YYYY")
     });
 
-    return new Promise((resolve, reject) => {
-        pdf.create(result, options).toBuffer((err, buffer) => {
-            if (err) {
-                reject(err);
-            } else {
-                resolve(buffer);
-            }
-        });
+    const ASSETS = __dirname + "/../../../../assets";
+    return await generatePDFFromHTMLString(result, {
+        includePaths: [
+            path.resolve(ASSETS, "creiners.png"),
```

Comment:
As an alternative to manually including all those files (and thus adding the empty test files to the assets in this repository) we could also think about simply including the entire assets directory and adding `<base href="./assets/">` to the certificates templates. Having those empty files will also complicate the current deployment process because this is basically a `git pull` which then, on every deployment, overwrites the actual assets files with those empty files.

Reply:
@geroembser So using `path.resolve(ASSETS)` copies all the files in an `assets/` subfolder? Then that makes a lot of sense